app/validators/policy_validator.rb

```diff
@@ -0,0 +1,13 @@
+class PolicyValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    begin
+      policy = ApiClient.send(options[:method], value)
+    rescue GetIntoTeachingApiClient::ApiError => e
```

Comment:
This should probably be removed if there's no clarity on how to handle an error?

Currently this will silently fail which is probably the more problematic outcome. For now I think its better to let the exception propogate, the app will trigger a 500 and if it becomes a specific problem, we can handle it then.

There are 2 likely categories of error
1) Api is down, in which case there's no point progressing things anyway
2) Intermittent network glitches, these should be handled in the api client itself

The client I'd originally written handled retries and my plan is to enhance the new no one with the same behaviour, thats not needed for the beta though.